File: src/test/java/student/StudentTest.java
package student;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void highestTest(){
        Student student = new Student("Courtney", 60, 98);
        Assertions.assertEquals(98,student.highest());
    }
    @Test
    public void getAverageTest(){
        Student student = new Student("Courtney", 60, 98);
        Assertions.assertEquals(79, student.getAverage());
    }
}

File: src/test/kotlin/com/courtney/StudentTest.kt
package com.courtney

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun highestTest() {
        val student = Student("Courtney", 88, 50)
        Assertions.assertEquals(88, student.highest())
    }

    @Test
    fun getAverageTest() {
        val student = Student("Courtney", 88, 50)
        Assertions.assertEquals((88+50)/2, student.getAverage())
    }

    @Test
    fun gradingTest() {
        val student = Student("Courtney", 88, 50)
        Assertions.assertEquals('C', student.grading())
    }

}